filetracker/migration_test.py

```diff
@@ -0,0 +1,111 @@
+"""Integration tests for client-server interaction."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import shutil
+import signal
+import tempfile
+import time
+import unittest
+from wsgiref.simple_server import make_server
+
+from filetracker.client import Client, FiletrackerError
+from filetracker.servers.migration import MigrationFileTrackerServer
+from filetracker.servers.files import FileTrackerServer
+
+_TEST_PORT_NUMBER = 45735
+
+
+class InteractionTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.cache_dir1 = tempfile.mkdtemp()
+        cls.cache_dir2 = tempfile.mkdtemp()
+        cls.server_dir = tempfile.mkdtemp()
+        cls.base_server_dir = tempfile.mkdtemp()
+        cls.temp_dir = tempfile.mkdtemp()
+
+        cls.base_server = FileTrackerServer(cls.base_server_dir)
+        base_port = _TEST_PORT_NUMBER + 10
```

Comment:
I think two constant ports would be clearer to read and debug.

Reply:
Certainly.